packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -414,8 +427,12 @@ class MonacoUtils {
    */
   static removeKeybinding(
     editor: monaco.editor.IStandaloneCodeEditor,
-    keybinding: string
+    keybinding: string | undefined
```

Comment:
Why not just make optional
```suggestion
    keybinding?: string
```

Reply:
The method makes no sense to call without this parameter, but the value could be undefined (Windows shortcut we remove, but not a mac shortcut).

I think it's better to have required params that can be undefined if the method makes 0 sense to call without a param